src/main/java/homework6/task3/AnyObjectDeepCloneApp.java

```diff
@@ -0,0 +1,36 @@
+package homework6.task3;
+
+import homework6.task2.CarCloneApp;
+
+/*реализуйте метод deepClone который принимает объект любого класса и возвращает глубокую копию этого класса*/
+public class AnyObjectDeepCloneApp implements Cloneable {
+
+    private Object inputObject;
+
+    public AnyObjectDeepCloneApp(Object inputObject) {
+        this.inputObject = inputObject;
+    }
+
+    public Object getInputObject() {
+        return inputObject;
+    }
+
+    public void setInputObject(Object inputObject) {
+        this.inputObject = inputObject;
+    }
+
+    @Override
+    public Object clone() throws CloneNotSupportedException {
```

Comment:
this method doesn't work correctly
it doesn't make deep clone
and for this purpose better use generics